apps/api/src/app/events/usecases/send-message/send-message-in-app.usecase.ts

```diff
@@ -68,26 +70,41 @@ export class SendMessageInApp extends SendMessageBase {
     const { actor } = command.step.template;
 
     if (actor && actor.type !== ActorTypeEnum.NONE) {
-      actor.data = await this.processAvatar(actor, command, notification);
+      actor.data = await this.processAvatar(actor, command);
     }
 
+    const organization = await this.organizationRepository.findById(command.organizationId);
```

Comment:
What do you think about using the `GetOrganization` use case from the Organization module and bring it as a dependency to the Events module? So we can start to isolate the DAL dependencies in their corresponding module (Organization in this case)?